rendezvous.ks

```diff
@@ -30,7 +30,7 @@ if target:position:mag > 25000 and approachX > 25000 {
 
   run node_hoh.
 
-  if utilHasNextNode() = false {
+  until utilHasNextNode() {
```

Comment:
kOS _finally_ added a function for this, so we can remove the util hack if convenient:
https://ksp-kos.github.io/KOS/structures/vessels/node.html?highlight=nextnode#global:HASNODE

Reply:
Done in my local version